nevergrad/benchmark/plotting.py

```diff
@@ -187,6 +188,19 @@ def create_plots(df: pd.DataFrame, output_folder: PathLike, max_combsize: int =
             # save
             name = "fight_" + ",".join("{}{}".format(x, y) for x, y in zip(fixed, case)) + ".png"
             name = "fight_all.png" if name == "fight_.png" else name
+            try:
+                if name == "fight_all.png":
+                    with open(str(output_folder / name) + ".cp.txt", "w") as f:
+                       f.write("ranking:\n")
+                       for i, algo in enumerate(data_df.columns[:8]):
+                           f.write(f"  algo {i}: {algo}\n")
+            except:
+                pass
+            name=re.sub(r'\([^()]*\)', '', name)
+            name=re.sub(r'\([^()]*\)', '', name)
+            name=re.sub(r'\([^()]*\)', '', name)
```

Comment:
This removes non-critical things in the name.

Reply:
it may be critical, because then names may become the same for 2 different xps, hence overriding an existing file
Also, avoid duplication, if you want to repeat 3 times, use a for loop ;)